go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/genometools/genometools
+
+go 1.15
```

Comment:
Does that mean that the code here requires Go 1.15? If so, is there a reason for such a specific version?

Reply:
`go mod init` automatically writes its own Go version in there.

https://golang.org/ref/mod#go gives you the meaning (link seems to be a bit broken, search for "go directive" on that page).

It denotes the expected version, but I don't see a reason why it wouldn't compile at least a few versions back. I just tried it with Go 1.12.17 and it works.

I'm not sure if people change that to earlier versions to document it more clearly what the expected Go version is, haven't looked for that in the wild. I want the CGI scripts to run on OpenBSD and that is still on 1.13.9. We could leave it as it is or go back to `go 1.13` here, for example.

What do you think?

Reply:
I see. No that's fine, since I don't use modules myself I just wasn't sure about the implications of having the version in there.